calendar-app/Components/GroupSection/InGroupSection.jsx

```diff
@@ -42,26 +43,47 @@ const GroupInput = ({ onRefresh }) => {
 
   const handleSendInvite = async () => {
     if (!username) {
-      Alert.alert('Please enter a username');
+      Toast.show({
+        type: 'error',
+        text1: 'Error',
+        text2: 'Please enter a username',
+      });
```

Comment:
you should use helper functions to reduce edundancy and make the code cleaner.

Reply:
Toast.show() is already a helper function